i18n/locales/en.json

```
@@ -78,5 +78,9 @@
   "components.api.apiChanges.addedIn": "Added in: {version}",
   "components.api.apiChanges.history": "History",
   "components.api.apiChanges.history.version": "Version",
-  "components.api.apiChanges.history.changes": "Changes"
+  "components.api.apiChanges.history.changes": "Changes",
+  "components.hero.downloadLts": "Download Node (LTS)",
```

```suggestion
  "components.home.hero.downloadLts": "Download Node (LTS)",
```

Please add the same for the following keys

---

done